packages/styles/components/Progress.module.scss

```diff
@@ -0,0 +1,82 @@
+@import '~@venice/styles/core';
+
+$colors: 'primary' var(--primary-color), 'secondary' var(--secondary-color),
+  'success' var(--success-color), 'danger' var(--danger-color);
+
+@mixin bar-progress {
+  background: var(--disabled-color-contrast);
+  background-image: linear-gradient(90deg, transparent 50%, currentColor 50%);
+  background-size: 10% 100px;
+  color: var(--disabled-color-contrast);
+}
+
+@mixin bar-progress-value {
+  background: var(--secondary-color);
+  transition: width var(--transition-duration) ease-in-out;
```

Comment:
we can use `scaleX` here to improve performance of CSS animation 😊

you [can check an example here](https://codepen.io/felipefialho/pen/zYOzMJz).

Reply:
But this isn't native progress bar of html